SPECS/curl/curl.spec

```diff
@@ -1,13 +1,15 @@
 Summary:        An URL retrieval utility and library
 Name:           curl
 Version:        7.76.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/NetworkingLibraries
 URL:            https://curl.haxx.se
 Source0:        https://curl.haxx.se/download/%{name}-%{version}.tar.gz
+Patch0:         CVE-2021-22898.patch
+#Patch1:         CVE-2021-22901.patch
```

Comment:
I think you missed uncommenting this line.

Reply:
yikes...forgot to save it.  good call.